.gitpod.yml

```diff
@@ -0,0 +1,13 @@
+ports:
+- port: 3000
+  onOpen: open-preview
+tasks:
+- init: yarn
+  command: yarn dev
+- openMode: split-right
+  command: echo -e "If you made any changes to this project enter your Github username below and add yourself as a contributor\n" &&
```

Comment:
If you just run `yarn contrib:add` on its own, it will ask the user for their username and contributions in a really pleasant manner.

Reply:
Thanks for telling me the better way to do it. I made the required changes, I hope you are good to go.